roles/gateway_update_otk_hostnames/README.md

```diff
@@ -0,0 +1,52 @@
+Update Hostnames used by OTK
+============================
+
+This role updates the hostnames used by OTK using the values configured under this role's `vars/main.yml`:
+- Gateway cluster.hostname cluster property
+- OTK's JDBC connection URLs
+- Callback URLs
+
+Requirements
+------------
+
+Before running this role:
+- OTK should already be migrated to the destination database, which is the internal MySQL database running on a Gateway.
+- Ensure the Ansible controller can modify the OTK database
```

Comment:
Instead of "Ensure the Ansible controller can modify the OTK database", say something like and also follow the Preupgrade analyzer readme related to granting of permissions via MYSQL (I know it is mentioned on L20 but need to be more specific) :

Perform the following steps to ensure that the Ansible controller can modify the OTK database:
 - configure the Group and Role variables below. 
 - execute the mysql command:
     CREATE USER @'localhost' IDENTIFIED BY ''; GRANT SELECT, INSERT, DELETE, UPDATE ON .* TO @'localhost';

  Make sure to revoke the privileges after the tasks is completed
      mysql xyz